src/client.rs

```diff
@@ -83,15 +122,15 @@ use crate::{
 /// equal to calling `connect()` function.
 ///
 /// The URL may be either ws:// or wss://.
-/// To support wss:// URLs, feature "tls" must be turned on.
+/// To support wss:// URLs, feature `native-tls` or `rustls-tls` must be turned on.
 ///
 /// This function "just works" for those who wants a simple blocking solution
 /// similar to `std::net::TcpStream`. If you want a non-blocking or other
 /// custom stream, call `client` instead.
 ///
-/// This function uses `native_tls` to do TLS. If you want to use other TLS libraries,
-/// use `client` instead. There is no need to enable the "tls" feature if you don't call
-/// `connect` since it's the only function that uses native_tls.
+/// This function uses `native_tls` or `rustls` to do TLS depending on the feature flags enabled. If
+/// you want to use other TLS libraries, use `client` instead. There is no need to enable the "tls"
+/// feature if you don't call `connect` since it's the only function that uses native_tls or rustls.
```

Comment:
Nitty clarification (feel free to adapt):
```suggestion
/// you want to use other TLS libraries, use `client` instead. There is no need to enable any of
/// the `*-tls` features if you don't call `connect` since it's the only function that uses them.
```
Once more occurrence below, as we have some docs duplication.

Reply:
Thanks for the suggestion. That is much clearer 🙇‍♂️